samples/crypto/psa_tls/src/psa_tls_functions_client.c

```diff
@@ -54,6 +58,7 @@ static int setup_tls_client_socket(void)
 
 	err = setsockopt(sock, SOL_TLS, TLS_SEC_TAG_LIST, sec_tag_list,
 			 sizeof(sec_tag_list));
+
```

Comment:
Intended?

Reply:
Pdk support will be properly removed in a future PR from what I understood